packages/loader/driver-utils/src/network.ts

```diff
@@ -29,19 +29,33 @@ export function isOnline(): OnlineStatus {
     return OnlineStatus.Unknown;
 }
 
+/** Base class for driver errors that standardizes adding driverVersion for logging */
```

Comment:
Alternative approach if we don't want to export this class is a function like this:

```
function addDriverVersion(error: IFluidErrorBase, driverVersion: string) {
    error.addTelemetryProperties({ driverVersion });
}
```
And then just call `addDriverVersions(this, driverVersion)` after the `super` call in each constructor.  🤷‍♂️